internal/oidc/verifier_test.go

```
@@ -369,3 +379,410 @@ func makeJWTHeader(header map[string]any) string {
+	if err != nil {
+		t.Fatalf("signing: %v", err)
+	}
+	return signingInput + "." + base64.RawURLEncoding.EncodeToString(sig)
+}
+
+func newOIDCTestServer(t *testing.T, key *rsa.PrivateKey, kid string) *httptest.Server {
+	t.Helper()
+	srv := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.URL.Path {
+		case "/.well-known/openid-configuration":
+			fmt.Fprintf(w, `{"issuer":"https://a.example.com","jwks_uri":"https://%s/openid/v1/jwks"}`, r.Host)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)